packages/airnode-node/src/evm/requests/blocking.ts

```diff
@@ -32,21 +32,16 @@ export function blockRequestsWithWithdrawals([
       return { ...acc, apiCalls: [...acc.apiCalls, apiCall] };
     }
 
+    // Drop request, but log that it is being ignored
     const warningLog = logger.pend(
       'WARN',
       `Ignoring Request ID:${apiCall.id} as it has a pending Withdrawal ID:${pendingWithdrawal.id}`
     );
-    const blockedCall: Request<ApiCall> = {
-      ...apiCall,
-      status: RequestStatus.Ignored,
-      // TODO: These error messages should be functions accepting id parameter and creating the error string
-      errorMessage: `${RequestErrorMessage.PendingWithdrawal}: ${pendingWithdrawal.id}`,
-    };
 
     return {
       ...acc,
       logs: [...acc.logs, warningLog],
-      apiCalls: [...acc.apiCalls, blockedCall],
+      apiCalls: [...acc.apiCalls],
```

Comment:
```suggestion
      apiCalls,
```

Reply:
Reverting as the suggestion fails CI

Reply:
Does it? It probably should be `apiCalls: acc.apiCalls` but destrucutre is not needed.

Reply:
Here is the failure: https://github.com/api3dao/airnode/runs/5351085862?check_suite_focus=true#step:6:133

But actually since the accumulator isn't being updated the whole line can be deleted 😄